src/js/header.js

```diff
@@ -0,0 +1,38 @@
+fetch('./html/header.html')
+  .then(response => response.text())
+  .then(data => {
+    document.getElementById('header').innerHTML = data;
```

Comment:
rzuca błędem - nie ma nic z id header. Jak zmieniam na queryselector, to sypie się stylowanie